phy/cluster/manual/tests/test_view_model.py

```diff
@@ -54,19 +72,20 @@ def _test_view_model(view_model_class, stop=True, **kwargs):
     show_test_start(vm.view)
     show_test_run(vm.view, _N_FRAMES)
 
-    # Merge the clusters and update the view.
-    debug("Merging.")
-    up = clustering.merge(clusters)
-    vm.on_select(up.added)
-    show_test_run(vm.view, _N_FRAMES)
-
-    # Split some spikes and update the view.
-    debug("Splitting.")
-    spikes = spikes[::2]
-    up = clustering.assign(spikes, np.random.randint(low=0, high=5,
-                                                     size=len(spikes)))
-    vm.on_select(up.added)
-    show_test_run(vm.view, _N_FRAMES)
+    if do_cluster:
+        # Merge the clusters and update the view.
+        debug("Merging.")
+        up = clustering.merge(clusters)
+        vm.on_select(up.added)
+        show_test_run(vm.view, _N_FRAMES)
+
+        # Split some spikes and update the view.
+        debug("Splitting.")
+        spikes = spikes[::2]
+        up = clustering.assign(spikes, np.random.randint(low=0, high=5,
```

Comment:
* we can use a fixed seed
* it is nice to have some randomness in a few specific tests, but in this case we have to make sure that they are completely robust